imports/filters/filters.go

```diff
@@ -140,6 +143,7 @@ func projectImportPath() string {
 		log.Fatal(errors.New("$GOPATH not set"))
 	}
 
-	// leverage the $GOPATH to strip out everything but the base git URL
-	return path[len(gopath+"/src/"):]
+	// trim the $GOPATH/src off the current working directory
+	gosrc := filepath.Join(gopath, "src") + string(filepath.Separator)
+	return strings.TrimPrefix(cwd, gosrc)
```

Comment:
Interesting point.  We can coerce it by using filepath.Split followed by
strings.Join.
On Mar 23, 2016 8:06 AM, "Jack Spirou" <notifications@github.com> wrote:

> In imports/filters/filters.go
> <https://github.com/govend/govend/pull/43#discussion_r57174899>:
>
> > @@ -140,6 +143,7 @@ func projectImportPath() string {
> >  		log.Fatal(errors.New("$GOPATH not set"))
> >  	}
> >
> > -	// leverage the $GOPATH to strip out everything but the base git URL
> > -	return path[len(gopath+"/src/"):]
> > +	// trim the $GOPATH/src off the current working directory
> > +	gosrc := filepath.Join(gopath, "src") + string(filepath.Separator)
> > +	return strings.TrimPrefix(cwd, gosrc)
>
> Quick question, for the projectImportPath method, I think we always want
> to return a unix like path (since import paths are always unixy), never a
> windows like path. I think this code will return a windows like path if the
> user is on windows, and that might mess up the Local filter method.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/govend/govend/pull/43/files/f68c5c2afeac4153624199576f16a85c359aca32#r57174899>
>